conf/webpack.config.js

```diff
@@ -2,23 +2,44 @@
 //bundle the web version of @openpgp/web-stream-tools for content script
 const path = require('path');
 
-module.exports = {
-  mode: 'production',
-  entry: {
-    main: '../build/generic-extension-wip/lib/streams/streams.js',
-  },
-  output: {
-    library: {
-      name: 'Stream',
-      type: 'var',
+module.exports = [
+  {
+    mode: 'production',
+    entry: {
+      main: '../build/generic-extension-wip/lib/streams/streams.js',
+    },
+    output: {
+      library: {
+        name: 'Stream',
+        type: 'var',
+      },
+      path: path.resolve('../build/generic-extension-wip/lib'),
+      filename: 'streams_web.js', // <--- Will be compiled to this single file
+    },
+    resolve: {
+      fallback: {
+        stream: false,
+      },
+      extensions: ['.js'],
     },
-    path: path.resolve('../build/generic-extension-wip/lib'),
-    filename: 'streams_web.js', // <--- Will be compiled to this single file
   },
-  resolve: {
-    fallback: {
-      stream: false,
+  {
+    mode: 'production',
+    entry: {
+      main: '../build/generic-extension-wip/lib/streams/streams.js',
+    },
+    output: {
+      library: {
+        type: 'commonjs',
+      },
+      path: path.resolve('../build/generic-extension-wip/lib'),
+      filename: 'streams_common.js', // <--- Will be compiled to this single file
+    },
+    resolve: {
+      fallback: {
+        stream: false,
+      },
+      extensions: ['.js'],
     },
```

Comment:
Previous build, streams_WEB.js decleared global `Stream` var variable and can't be used in tests as we used import style.

Reply:
By the way, maybe it's possible to use built-in browser's Streams API instead of this lib - https://developer.mozilla.org/en-US/docs/Web/API/Streams_API? It's supported in both Chrome and Firefox

Reply:
Let me check.

Reply:
We are using @openpgp/web-stream-tools library for streaming and native `Streams Api` doesn't support functions such as readToEnd which is supported by @openpgp/web-stream-tools  and used in our project.

We just need to implement our own `readToEnd` function.

Maybe we can completely get rid of `@openpgp/web-stream-tools`.

Do you want me to try this?

Also do you wanna me to do this in PR or in other PR?

Reply:
Let's try it in a separate PR, thanks!